neo/Consensus/ConsensusService.cs

```diff
@@ -520,7 +520,7 @@ private void SendPrepareRequest()
                 foreach (InvPayload payload in InvPayload.CreateGroup(InventoryType.TX, context.TransactionHashes.Skip(1).ToArray()))
                     localNode.Tell(Message.Create("inv", payload));
             }
-            ChangeTimer(TimeSpan.FromSeconds(Blockchain.SecondsPerBlock << (context.ViewNumber + 1)));
+            ChangeTimer(TimeSpan.FromSeconds((Blockchain.SecondsPerBlock << (context.ViewNumber + 1)) - Blockchain.SecondsPerBlock));
```

Comment:
I think it should subtract `Blockchain.SecondsPerBlock` only if `context.ViewNumber == 0`.

Reply:
makes sense, Erik, eagle eyes as always.
I will soon take a look to be sure about it.

Reply:
Exactly, Erik. Let's add this filter.

Can it be done inline @igormcoelho?